lib/helpers.js

```diff
@@ -26,6 +26,17 @@ _beforeEach.withApp = function(app) {
   });
 }
 
+_beforeEach.cleanDatasource = function(dsName) {
+  beforeEach(function() {
+    if(!dsName) dsName = 'db';
+
+    if (this.app) {
+      this.app.datasources[dsName].automigrate();
```

Comment:
Thanks for the catch, didn't think about that. Originally this was after which theoretically would work, but still fail on the next describe...